### PR TITLE
Pull Requests template

### PR DESCRIPTION
**Pull request by carlacfw**
Pull requests template documentation for pull requests guidelines.
#6

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Code style
+
+When submitting code, please make every effort to follow existing conventions and
+style in order to keep the code as readable as possible. 
+
+## Testing
+
+We kindly ask that all new features and fixes for an issue should include any unit tests.
+Even if it is small improvement, adding a unit test will help to ensure no regressions or the issue is not re-introduced.
```
Comment (ojongerius) [inline .github/CONTRIBUTING.md]: even if it is **a** small improvement?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Contributing to FreeCodeCamp Wellington
+
+If you would like to contribute code you can do so through GitHub by forking the repository and sending a pull request on a branch other than the master branch.
+
+## Pull Requests
+
+Pull requests eventually need to resolve to a single commit. The commit log should be easy to read as a change log. We use the following form to accomplish that.
+* First line is a <=72 character description in present tense, explaining what this does.
+  * Ex. "Fixes regression on encoding vnd headers" > "Fixed encoding bug", which forces the reader to look at code to understand impact.
```
Comment (ojongerius) [inline .github/PULL_REQUEST_TEMPLATE.md]: on encoding **and** headers?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* Do not include issue links in the first line as that makes pull requests look weird.
+  * Ex. "Addresses #345" becomes a pull request title: "Addresses #345 #346"
+* After the first line, use markdown to concisely summarize the implementation.
+  * This isn't in leiu of comments, and it assumes the reader isn't intimately familar with code structure.
+* If the change closes an issue, note that at the end of the commit description ex. "Fixes #345"
+  * GitHub will automatically close change with this syntax.
```
Comment (ojongerius) [inline .github/PULL_REQUEST_TEMPLATE.md]: will automatically close **changes**?

File: .github/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Firmware Version:
```
Comment (ojongerius) [inline .github/issue_template.md]: Firmware version is not relevant to us

File: .github/issue_template.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* Operating System:
+* SDK version:
```
Comment (ojongerius) [inline .github/issue_template.md]: SDK not relevant to us either

File: .github/issue_template.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* Toolchain version:
```
Comment (ojongerius) [inline .github/issue_template.md]: Toolchain is not applicable

**Review by ojongerius — CHANGES_REQUESTED** [review]
Awesome, thanks @carlacfw 🎉

**Comment by ojongerius** [comment]
Just got back from holidays 😄 Thanks @carlacfw!

**Comment by carlacfw** [comment]
Cool, hope you have had a great holiday! Welcome back then haha

**Pull request merged**